### DISCUSSION TITLE
Copilot auth token was valid, but does not contain 'chat' field.

### DISCUSSION THREAD

**Discussion by jcipsen** [community/community#54893] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have been invited by my organization to use Copilot; however, when I start Visual Studio 2022, I encounter the following error:

"**Copilot auth token was valid, but does not contain 'chat' field**."

Copilot is installed and enabled, and I am logged in to both GitHub and my organization's DevOps account.

I would appreciate any advice on resolving this issue, as I am currently unable to use Copilot.

Thank you in advance for your assistance.
Jutta

**Comment by iripple** [comment]
it's the same thing as me. Are you using the ARM version?

**Comment by jcipsen** [comment, marked as answer]
We found the issue. Make sure to uninstall  _Visual Studio Extension for GitHub Copilot Preview_ and install **GitHub Copilot** instead

**Reply by x600** [reply]
Where to download GitHub Copilot? It doesn't seem to be appearing in Visial Studio extensions search?
